### ISSUE TITLE
Form errors on collections / subforms

### ISSUE THREAD

**Issue by Flippsus** [symfony/symfony#20459]
In my twig template form validation errors are shown, when using `{{ form_errors(form) }}` or  `{{ form_errors(form.fieldname) }}` in general.

If the field is of type Collection, `form_errors(form)` shows the validation error, e.g. the collection has to be of min 1 element ..., but `form_errors(form.the_collection_field) }}` does **not**. It outputs an empty string.

Anyone else had this issue?

**Comment by peterrehm** [comment]
@Flippsus Look up error bubbling http://symfony.com/doc/current/reference/forms/types/collection.html#error-bubbling. That should solve your issue.

**Comment by peterrehm** [comment]
@Flippsus Did you get your issue solved?

**Comment by Flippsus** [comment]
I think I lost my concentration on it. I just ignored the behaviour, in order to get back to it later. In the meantime I had to implement another rendering algorithm (for a dynamically created form). The problem did not occure anymore. I am not sure, what the real reason was. Thank you, for the hint with error_bubbling. I will comment back in this thread, if I get to know something new. Let's just forget about this Notbug for now. Thank you.
